models.py

```diff
@@ -791,11 +791,13 @@ def get_chronological(
                            'cronorder', limit, version)
 
     keys = get_chunk_memcache_keys(Feature.all(), KEY)
-    feature_list = memcache.get_multi(keys)
+    feature_list = ramcache.get_multi(keys)
+    logging.info('getting chronological feature list')
```

Comment:
I have no objections to this logging statement, but wanted to make sure you intended to leave that there.

Reply:
Yeah, I'd like to leave it in for now.  If something goes wrong, having more stuff logged will make investigation easier.